ci-operator/step-registry/ipi/conf/gcp/xpn/ipi-conf-gcp-xpn-commands.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+CONFIG="${SHARED_DIR}/install-config.yaml"
+PATCH=/tmp/install-config-xpn.yaml.patch
+
+HOST_PROJECT="openshift-dev-installer"
+HOST_PROJECT_CONTROL_SUBNET="installer-shared-vpc-subnet-1"
+HOST_PROJECT_COMPUTE_SUBNET="installer-shared-vpc-subnet-2"
+HOST_PROJECT_NETWORK="installer-shared-vpc"
+
+cat > "${PATCH}" << EOF
+featureSet: TechPreviewNoUpgrade
+credentialsMode: Passthrough
+platform:
+  gcp:
+    computeSubnet: ${HOST_PROJECT_COMPUTE_SUBNET}
+    controlPlaneSubnet: ${HOST_PROJECT_CONTROL_SUBNET}
+    createFirewallRules: Disabled
+    network: ${HOST_PROJECT_NETWORK}
+    networkProjectID: ${HOST_PROJECT}
+        publicDNSZone:
```

Comment:
```
    publicDNSZone:
      id: installer-public-zone
      project: {HOST_PROJECT}
```

Try this, looks like the alignment is off. The public zone information should be in line with the other variables.